### ISSUE TITLE
can a KeyDB instance be replicaof redis-aerver?

### ISSUE THREAD

**Issue by Nadav42** [Snapchat/KeyDB#726]
Is it possible to set up KeyDB instances to be read-only replicas of pure redis server?

does keydb improve on the replicaof functionality? Does it benefit from keydb multithreading or something?

**Comment by msotheeswaran-sc** [comment]
Yes you can make a keydb instance replicate a redis instance, the multithreading will not affect replication, but should provider better read throughput compared to a redis instance.
